src/lib/locale.store.ts

```diff
@@ -0,0 +1,63 @@
+import { readable, writable, type Readable, type Writable } from 'svelte/store';
+import { z } from 'zod';
+
+import * as fallback from '$locales/us/en/index.json';
+
+const stringsSchema = z.object({
+  alreadyHaveAnAccount: z.string(),
+  dontHaveAnAccount: z.string(),
+  closeModal: z.string(),
+  chooseDifferentUsername: z.string(),
+  ensurePasswordIsMoreThan: z.string(),
+  ensurePasswordHasOne: z.string(),
+  givenName: z.string(),
+  loading: z.string(),
+  loginButton: z.string(),
+  loginHeader: z.string(),
+  loginSuccess: z.string(),
+  mail: z.string(),
+  NameCallback: z.string(),
+  nextButton: z.string(),
+  PasswordCallback: z.string(),
+  pleaseCheckValue: z.string(),
+  preferencesmarketing: z.string(),
+  preferencesupdates: z.string(),
+  registerButton: z.string(),
+  registerHeader: z.string(),
+  registerSuccess: z.string(),
+  securityAnswer: z.string(),
+  securityQuestions: z.string(),
+  securityQuestionsPrompt: z.string(),
+  showPassword: z.string(),
+  sn: z.string(),
+  submitButton: z.string(),
+  successMessage: z.string(),
+  termsAndConditions: z.string(),
+  useValidEmail: z.string(),
+  ValidatedCreatePasswordCallback: z.string(),
+  ValidatedCreateUsernameCallback: z.string(),
+});
```

Comment:
This is Zod. You can check it out here: https://zod.dev. Let me know what you think about it. I really like its use for runtime object schema validation and its ability to generate TypeScript types from these schemas. This will be really important when this eventually gets used as a framework. It will give the devs immediate feedback on config objects, content objects ... anything that has an important schema.